docs/content/en/integrations.md

```diff
@@ -284,3 +284,22 @@ We recommend to take a look at this tutorial made by [Pascal Cauhépé](https://
 👉 &nbsp;https://nuxt-content-and-forestry.netlify.app
 
 </alert>
+
+## @nuxtjs/i18n
+
+You can use Nuxt Content with [nuxt-i18n](https://github.com/nuxt-community/i18n-module).
```

Comment:
I realize it's inconsistent in this document, but it would be nice to include just a bit more of an explanation of what the i18n module is for.

```suggestion
You can use Nuxt Content with [nuxt-i18n](https://github.com/nuxt-community/i18n-module) to support multiple languages.
```